frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx

```diff
@@ -11,110 +11,111 @@ import {
   ChartTooltip,
 } from '@patternfly/react-charts';
 import { Tooltip } from '@patternfly/react-core';
-import { DataPoint } from '@console/internal/components/graphs';
 import { K8sKind, referenceForModel } from '@console/internal/module/k8s';
 import { resourcePathFromModel } from '@console/internal/components/utils';
 import { CEPH_STORAGE_NAMESPACE } from '@console/ceph-storage-plugin/src/constants';
 import { getBarRadius, StackDataPoint } from './utils';
 import { OTHER, CLUSTERWIDE, BUCKETCLASSKIND } from './consts';
 import './breakdown-card.scss';
 
-const LinkableLegend: React.FC<LinkableLegendProps> = React.memo((props: LinkableLegendProps) => {
-  const { metricModel, datum, ocsVersion } = props;
-  let href: string = resourcePathFromModel(metricModel, datum.link, datum.ns);
-  const customLegend = (
-    <Tooltip content={datum.link} enableFlip>
-      <ChartLabel
-        {...props}
-        lineHeight={1.2}
-        style={[
-          { ...datum.labels, fontSize: 9 },
-          { fill: 'black', fontSize: 8 },
-        ]}
-      />
-    </Tooltip>
-  );
-  if (datum.name[0] === OTHER || datum.name[0] === CLUSTERWIDE) {
-    return customLegend;
-  }
-  if (metricModel.kind === BUCKETCLASSKIND) {
-    if (ocsVersion) {
-      href = `/k8s/ns/${CEPH_STORAGE_NAMESPACE}/clusterserviceversions/${ocsVersion}/${referenceForModel(
-        metricModel,
-      )}/${datum.link}`;
-    } else {
+export const LinkableLegend: React.FC<LinkableLegendProps> = React.memo(
+  (props: LinkableLegendProps) => {
+    const { metricModel, datum, ocsVersion } = props;
+    let href: string = resourcePathFromModel(metricModel, datum.link, datum.ns);
+    const customLegend = (
+      <Tooltip content={datum.link} enableFlip>
+        <ChartLabel
+          {...props}
+          lineHeight={1.2}
+          style={[
+            { ...datum.labels, fontSize: 9 },
+            { fill: 'black', fontSize: 8 },
+          ]}
+        />
+      </Tooltip>
+    );
+    if (datum.name === OTHER || datum.name === CLUSTERWIDE) {
```

Comment:
```suggestion
    if (datum.name[0] === OTHER || datum.name[0] === CLUSTERWIDE) {
```

Reply:
https://github.com/openshift/console/pull/4852/files#diff-2b7e72b6b33459dbcc45bf9440b06f29L49
This tells a different story